ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -815,13 +815,13 @@ public class SwiftDeviceCalendarPlugin: NSObject, FlutterPlugin, EKEventViewDele
             ekEvent!.notes = description
             ekEvent!.isAllDay = isAllDay
             ekEvent!.startDate = startDate
-            if (isAllDay) { ekEvent!.endDate = startDate }
-            else {
-                ekEvent!.endDate = endDate
-
-                let timeZone = TimeZone(identifier: startTimeZoneString ?? TimeZone.current.identifier) ?? .current
+			ekEvent!.endDate = endDate
+            
+			if (!isAllDay) { 
+				let timeZone = TimeZone(identifier: startTimeZoneString ?? TimeZone.current.identifier) ?? .current
                 ekEvent!.timeZone = timeZone
-            }
+			}
+            
```

Comment:
The formatting seems a bit off, looks like the new lines have double the indentation. Please make sure to use spaces instead of tabs.

Reply:
Fixed formatting to spaces